portal/index.php

```diff
@@ -104,7 +104,12 @@
 <html>
 <head>
     <title><?php echo xlt('Patient Portal Login'); ?></title>
-    <?php Header::setupHeader(['datetime-picker']); ?>
+    <?php
+        $css = $GLOBALS['css_header'];
+        $GLOBALS['css_header'] = "";
+        Header::setupHeader(['datetime-picker']);
```

Comment:
change above work around to so following instead:
``````
<?php Header::setupHeader(['no_main-theme', 'datetime-picker']); ?>
``````
(do note this will only work if your branch inlcludes the recent change in codebase; could always bring in this code and then make these related changes after)

Reply:
(or just make the change and bring it into the codebase; I have faith it will work just fine :) )